pkg/workloads/reservation.go

```diff
@@ -268,6 +268,9 @@ func (a *API) setupPool(r *http.Request) (interface{}, mw.Response) {
 		filter = phonebook.UserFilter{}
 		filter = filter.WithID(schema.ID(reservation.SponsorTid))
 		sponsor, err := filter.Get(r.Context(), db)
+		if !sponsor.IsTrustedChannel {
+			return nil, mw.UnAuthorized(fmt.Errorf("the sponsor tid is not authorized"))
```

Comment:
can we please have the sponsor tid in the error message?

Reply:
done